### PR TITLE
List available cloud regions

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-5794

List of available regions.

### CODE REVIEW

**Comment by gemma-shay** [comment]
@emily-horing not super committed to putting these on the create pages, lmk if you think FAQ or somewhere else would be better.

File: cockroachcloud/create-your-cluster.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ To create a multi-region cluster, click **Add regions** until you have the desir
 
 {{site.data.alerts.callout_info}}
 Multi-region clusters must contain at least 3 regions to ensure that data spread across regions can survive the loss of one region. See [Planning your cluster](plan-your-cluster.html?filters=dedicated) for the requirements and recommendations for {{ site.data.products.dedicated }} cluster configuration.
-{{site.data.alerts.end}}
+{{site.data.alerts.end}}
```
Comment (rmloveland) [inline cockroachcloud/create-your-cluster.md:47]: extreme nit, but this adds spaces after the `}` which is unnecessary

also just ignore this comment if it's annoying lol

File: cockroachcloud/frequently-asked-questions.md [hunk 2]
```diff
@@ -17,7 +17,26 @@ This page answers the frequently asked questions about {{ site.data.products.ser
 
 ### In what clouds and regions is {{ site.data.products.db }} available?
 
-The  {{ site.data.products.db }} Console will always show the latest available cloud infrastructure providers and regions. Open the [**Create Cluster**](https://cockroachlabs.cloud/cluster/create) page and select a Plan to see which providers and regions are currently available.
+The following regions are available for {{ site.data.products.dedicated }}:
+
+GCP                                          | AWS
+---------------------------------------------|------
+Frankfurt, Germany (`europe-west3`)          | Osaka, Japan (`ap-northeast-3`)
```
Comment (rmloveland) [inline cockroachcloud/frequently-asked-questions.md:24]: Strongly suggest alphabetizing each of these lists for the readers' ease of finding their desired region

My sense is it's better to put the region name first and alphabetize by that e.g. `us-west1` (Oregon) but idk 

(unless there is some reason not to alphabetize I may not be aware of)

File: cockroachcloud/serverless-faqs.md [hunk 3]
```diff
@@ -39,6 +39,21 @@ With {{ site.data.products.serverless }}, you are charged for the storage and ac
 
 No, you can create a Serverless cluster that is free forever. If you choose to set a spend limit for your cluster, you will only be charged for the resources you use up to your spend limit.
 
+### What regions are available for {{ site.data.products.serverless }} clusters?
+
+The following regions are available for {{ site.data.products.serverless }}:
+
+GCP                                          | AWS
+---------------------------------------------|------
+Jurong West (`asia-southeast1`)              | Singapore (`ap-southeast-1`)
```
Comment (rmloveland) [inline cockroachcloud/serverless-faqs.md:48]: same comments here as above re: alphabetizing by region names

**Review by rmloveland — APPROVED** [review]
LGTM had some opinions about alphabetizing but that's it.  Nothing blocking

**Pull request merged**